v2/internal/controllers/to_azure_secrets_test.go

```diff
@@ -95,7 +99,7 @@ func Test_UserSecretInDifferentNamespace_SecretNotFound(t *testing.T) {
 	secretName := tc.Namer.GenerateName("admin-pw")
 	nameSpaceName := "secret-namespace"
 
-	createNamespaces(tc, nameSpaceName)
+	tc.CreateTestNamespaces(nameSpaceName)
```

Comment:
minor: I know you didn't write this but `namespace` is all one word so this should be `namespaceName`